pagopa/pn-frontend-e2e
# Call for notification with payment

## Short description

Implemented a call to create a notification with payment

## List of changes proposed in this pull request

- Improved the model class to create a JSON for the notifications with payment
- Implemented test 10394 that use the new call

## How to test

Simply run `mvn clean test -D"cucumber.filter.tags=@annullamentoNotifica"` and verify if the test works fine.
